helpers_test.go

```diff
@@ -0,0 +1,34 @@
+package libentitlement
+
+import (
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func testSpec() *specs.Spec {
```

Comment:
is this necessary given the same function exists in helpers.go?  We should remove one.

Reply:
[As discussed] I'll move this as well as *shareable* helper code for test purposes to a `testutil` package.